.gitignore

```diff
@@ -6,3 +6,5 @@ node_modules
 test
 npm-debug.log
 package-lock.json
+pnpm-lock.yaml
+yarn.lock
```

Comment:
the package lockfile might be getting exposed at some point: https://github.com/orgs/projectdysnomia/projects/1/views/1?sliceBy%5Bvalue%5D=next&pane=issue&itemId=87458206

Reply:
Should that be handled by a different PR or should I just remove this .gitignore change?

Reply:
Exposing it should be probably handled in a different PR along with a more official switch to pnpm :)